Project.toml

```diff
@@ -1,22 +1,24 @@
 name = "NMF"
 uuid = "6ef6ca0d-6ad7-5ff6-b225-e928bfa0a386"
 license = "MIT"
-version = "0.5.3"
+version = "1.0.0"
```

Comment:
> and that in turn requires a minor version bump (policy of the General registry).

Did you mean a major version bump? From 0.5 to 1.0 is a major bump (https://semver.org/).